packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -29,21 +29,29 @@ const getFocusedStyles = ({
   colors: StrictColorScheme<ItemType<typeof menuColorAreas>>;
 }): ICSSInJSStyle => {
   const { primary, underlined, active, vertical } = props;
+
   if (active && !underlined && !vertical) return {};
-  return {
-    color: v.colorActive,
-    background: v.backgroundColorFocus || colors.backgroundFocus,
 
-    ...(primary && {
-      color: colors.foregroundFocus,
+  return {
+    ...(!vertical && {
```

Comment:
Can we avoid there a condition for `!vertical`? I.e.

```
...vertical && {
  color: someValue,
  background: someValue,
}
```

Reply:
Thanks